src/main/java/frc/robot/subsystems/vision/NoteCamera.java

```diff
@@ -0,0 +1,50 @@
+package frc.robot.subsystems.vision;
+
+
+import org.littletonrobotics.junction.Logger;
+import org.photonvision.PhotonCamera;
+import org.photonvision.targeting.PhotonTrackedTarget;
+
+public class NoteCamera {
+
+  PhotonCamera camera;
+  String path;
+
+  private final double MIN_AREA = 10;
+
+  public NoteCamera(String name) {
+    // This is a constructor
+    camera = new PhotonCamera(name);
+    path = "Vision/" + name.replace("_", "");
+  }
+
+  public PhotonTrackedTarget getTarget() {
+    if(1==1) return null;
+    var res = camera.getLatestResult();
```

Comment:
please use a more specific data type here to match the return type of camera.getLatestResult()